src/frontend/hooks/useNavigateUtenUlagretSjekk.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import { useApp } from '../context/AppContext';
+
+/**
+ * Kan ikke kjøre navigate direkte fordi navigate er i blocked state,
+ * selv om nullstill ligger før denne metoden
+ */
+export const useNavigateUtenUlagretSjekk = () => {
+    const { nullstillIkkePersisterteKomponenter } = useApp();
+    const navigate = useNavigate();
+
+    const [path, settPath] = useState<string>();
+    useEffect(() => {
+        if (path) {
+            navigate(path);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Er det bedre å skru av denne linting-regelen dersom vi uansett "alltid" gjør unntak fra den?

Reply:
Syns det er greit at det kommer en påminnelse om at "her har du ikke tatt med alle" fordi det fort kan bli feil uten. Kan man få til å beholde det men at det er lov å commite med akk denne warningen? 

Hvis ikke tenker jeg at det er greit at man må ta et aktivt valg om å ignorere noe egt. Blir også tydligere for neste at her er det noe som vi bevisst ikke lytter på